### DISCUSSION TITLE
PNG to JPG and AVIF

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7101] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

I have thousands of png files that I want to convert into individual jpgs and avif files, both without losing quality. How do I do that?

**Comment by fmw42** [comment]
What was your convert command?  Why do you think you will lose quality?  JPG is already **lossy** compressed and you cannot get that quality back fully. PNG is **losslessly** compressed.

**Reply by Jay123210599** [reply]
Never mind jpegs, I don't know what the convert command for avif files. Can you tell me what that is?

**Comment by fmw42** [comment]
`magick image.png image.avif`
